notebook/01_tutorial.ipynb

```diff
@@ -47,9 +69,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import sys; sys.path.append('..')\n",
+    "import warnings; warnings.simplefilter(\"ignore\")\n",
+    "\n",
     "# load csv stock data\n",
     "task_csvdata = {\n",
-    "    'id': 'node_csvdata',\n",
+    "    'id': 'load_csv_data',\n",
```

Comment:
since we have the TaskSpecSchema class. change the 'id', 'type' ... to TaskSpecSchema.task_id, TaskSpecSchema.node_type

Reply:
Changed.